autorest/codegen/models/operation.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self.description = description
         self.url = url
         self.method = method
-        self.parameters = parameters
+        self.parameters = parameters if parameters else []
```

Comment:
To make it consistent, then do the same thing for line 38 to 43 (I like your neat approach)